py/README.md

```diff
@@ -53,7 +51,8 @@ See [oss.configu.com/py](https://oss.configu.com/py/configu.html)
 
 ### Requirements
 
-1. Follow the [Development](https://github.com/configu/configu/blob/main/CONTRIBUTING.md#development) section from the `CONTRIBUTING.md`.
+1. Follow the [Development](https://github.com/configu/configu/blob/main/CONTRIBUTING.md#development) section from
+   the `CONTRIBUTING.md`.
```

Comment:
why newline here?

Reply:
ide default. python project shouldn't have lines longer 79 - 119 chars. 
since it's not python, reverted to long line.